src/taco-utils/telemetry.ts

```diff
@@ -293,7 +296,7 @@ module TacoUtility {
                 logger.logLine();
 
                 var deferred: Q.Deferred<boolean> = Q.defer<boolean>();
-                var yesOrNoHandler = readline.createInterface({ input: process.stdin, output: process.stdout });
+                var yesOrNoHandler = readline.createInterface({ input: process.stdin, output: process.stdout, terminal: false });
```

Comment:
Just to be sure: We deliberately don't want to print out the values used when doing something like `taco feedback < answersFile`?